react-charts/bullet-chart/step1.md

```diff
@@ -1,10 +1,18 @@
-PatternFly React charts are based on the [Victory chart](https://formidable.com/open-source/victory/docs/victory-chart/) library, along with additional functionality, custom components, and theming for PatternFly. 
-This provides a collection of React based components you can use to build PatternFly patterns with consistent markup, styling, and behavior. 
-
-PatternFly React charts are composed using separate components, so features are more easily customizable. 
-In this course, we're going to build a PatternFly bullet chart together - starting with a simple chart, adding qualitative ranges, primary comparative measures, a comparative warning measure, tooltips, labels, a legend, and concluding by changing the theme color.
-You'll learn how to use PatternFly React chart components together to build a consistent user experience.
-
-The image below is what we'll end up with when we are done.
-
-<img src="bullet-chart/assets/final.png" alt="Completed chart component" style="box-shadow: rgba(3, 3, 3, 0.2) 0px 1.25px 2.5px 0px;" />
+Begin by adding a CSS selector to define the chart size.
+
+Katacoda is setting up a new React application. Begin coding once the server starts and "Welcome to PatternFly" is visible on the lower pane.
+
+1. <strong>Navigate to the `src` folder and open `src/app.css`{{open}}<strong>
+
+2. <strong>Add CSS to define the overall chart size.<strong>
+
+Copy the following code into the `app.css` file, replacing all the content:
+
+<pre class="file" data-filename="src/app.css" data-target="replace">
+.chart-container {
+  height: 150px;
+  width: 600px;
+}
+</pre>
+
+<strong>Note: </strong>Victory renders responsive containers, so the chart's width and height props do not determine the width and height of the chart in number of pixels. Instead, they define an aspect ratio for the chart. The exact number of pixels will depend on the size of the container the chart is rendered into. Typically, the parent container is set to the same width as the chart to maintain the aspect ratio and scale.
```

Comment:
This is formatted differently than the area chart tutorial. I assume area chart (below, on the right) is what we want to follow.

![Screen Shot 2019-12-09 at 10 34 32 AM](https://user-images.githubusercontent.com/17481322/70449260-b47cef00-1a6f-11ea-8aec-bec910c2e39d.png)

Reply:
Fixed